### PR TITLE
[Release Guide] Document Test Days

### PR DESCRIPTION
**Pull request by mkolesnik**
Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

File: src/content/development/contribution-guide/_index.en.md [hunk 1]
```diff
@@ -152,6 +152,30 @@ To be expanded
 
 ### Test Days
 
-To be expanded
+Test days are held in order to validate pre-released versions by the project members and the wider community.
+Any bugs opened on a test day will be triaged soon after.
+
+The goals of these days are:
```
Comment (Jaanki) [inline src/content/development/contribution-guide/_index.en.md:219]:
```suggestion
The goals of these days are to:
```

File: src/content/development/contribution-guide/_index.en.md [hunk 2]
```diff
@@ -152,6 +152,30 @@ To be expanded
...
+
+* Verify any new features that were introduced during the last [sprint](#sprints).
+* Validate any [bugs](#bugs) that were fixed were actually fixed.
+* Find any regressions in existing functionality.
+
+Test days will be led by one of the project members, who will be responsible for:
+
+* Creating a test day document, if one doesn't exist yet.
```
Comment (Jaanki) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
* Creating a test day spreadsheet, if one doesn't exist yet by Using the [test day template].
```

File: src/content/development/contribution-guide/_index.en.md [hunk 3]
```diff
@@ -152,6 +152,30 @@ To be expanded
...
+  * Using the [test day template].
```
Comment (Jaanki) [inline src/content/development/contribution-guide/_index.en.md]: Merged with above point.

**Review by Jaanki — CHANGES_REQUESTED** [review]

File: src/content/development/contribution-guide/_index.en.md (hunk 1 above)
Comment (dfarrell07) [reply src/content/development/contribution-guide/_index.en.md:219]: IMHO it's okay as-is

File: src/content/development/contribution-guide/_index.en.md [hunk 4]
```diff
@@ -152,6 +152,30 @@ To be expanded
...
 ### Test Days
 
-To be expanded
+Test days are held in order to validate pre-released versions by the project members and the wider community.
+Any bugs opened on a test day will be triaged soon after.
+
+The goals of these days are:
+
+* Verify any new features that were introduced during the last [sprint](#sprints).
+* Validate any [bugs](#bugs) that were fixed were actually fixed.
```
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
* Validate any [bugs](#bugs) that were closed were actually fixed.
```

**Review by dfarrell07 — APPROVED** [review]

**Review by maayanf24 — APPROVED** [review]

**Comment by dfarrell07** [comment]
@Jaanki Can you re-review?

**Review by Jaanki — APPROVED** [review]

**Pull request merged**